src/views/admin/AdminView.tsx

```diff
@@ -23,20 +22,59 @@ export const AdminView = () => {
       window.location.reload()
     }
   }
+
+  const handleExportClick = () => {
+    const storageKeys = Object.keys(window.localStorage)
+    const storage = storageKeys.reduce((acc, key) => {
+      const rawValue = window.localStorage.getItem(key)
+      if (rawValue) {
+        acc[key] = JSON.parse(rawValue)
+      }
+
+      return acc
+    }, {} as Record<string, unknown>)
+    const jsonStorage = JSON.stringify(storage)
+
+    const linkElement = document.createElement('a')
+    linkElement.setAttribute('href', 'data:application/json;charset=utf-8,' + encodeURIComponent(jsonStorage))
+    linkElement.setAttribute('download', 'atlas-export.json')
```

Comment:
Just an idea, but I think it could be useful - maybe we could add formatted date to the name of the file? For example `atlas-export-29-06-2021-1307.json`